src/index.html

```diff
@@ -1,12 +1,355 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
You need to add a Roboto font to the project. Use the correct link, please. Now san-serif is applied 
![image](https://github.com/mate-academy/layout_catalog/assets/99403198/fed9005a-92c8-499f-ab8c-f46593c2a337)